### PR TITLE
V10: 13125 don't return "null"

### PR DESCRIPTION
**Pull request by Zeegaan**
Fixes https://github.com/umbraco/Umbraco-CMS/issues/13125
# Notes
- Added null check to return null if parameter is null

# How to test
- Follow the detailed steps in issue 🙌

### CODE REVIEW

**Comment by elit0451** [comment]
Looks good, waiting for the pipelines' green light and will be merging 💪

**Comment by Zeegaan** [comment]
I have pointed this for v11, as this is a big behavioral change, where we now convert `null` values to empty strings, instead of `null`, so this needs some testing.

## Note
This might be backported to v10 if found not to be an issue after the v11rc testing period 🙌

File: src/Umbraco.Core/PropertyEditors/DataValueEditor.cs [hunk 1]
```diff
@@ -386,6 +386,11 @@ public virtual string ConvertDbToString(IPropertyType propertyType, object? valu
     /// <exception cref="System.ArgumentOutOfRangeException">ValueType was out of range.</exception>
     internal Attempt<object?> TryConvertValueToCrlType(object? value)
     {
+        if (value is null)
+        {
+            Attempt.Succeed<object?>(null);
```
Comment (elit0451) [inline src/Umbraco.Core/PropertyEditors/DataValueEditor.cs:391]:
```suggestion
            return Attempt.Succeed<object?>(null);
```

**Review by elit0451 — CHANGES_REQUESTED** [review]
Looks good, the only thing is that you missed the `return` 😅 without it the error is still there

**Comment by CarlSargunar** [comment]
Hi @Zeegaan - do you know if this is likely to now be back-ported to 10 as mentioned in earlier comments?

**Comment by Zeegaan** [comment]
Hi @CarlSargunar, I don't think we can, as this would be breaking. I know all bugs are on some level considered "behavioral breaking", but I could suspect someone relying on this behavior somewhere, and thus it would break their site 😅

**Pull request merged**